ci/do_ci.sh

```diff
@@ -8,22 +8,21 @@ set -e
 echo "building using ${NUM_CPUS} CPUs"
 
 function bazel_release_binary_build() {
-  cd "${ENVOY_CONSUMER_SRCDIR}"
   echo "Building..."
-  bazel --batch build ${BAZEL_BUILD_OPTIONS} -c opt @envoy//source/exe:envoy-static.stripped.stamped
+  cd "${ENVOY_BUILD_DIR}"
```

Comment:
We need it for `bazel.coverage` builds, due to the weirdness of the circular symlinks that Bazel creates where you build and `gcovr` not being able to cope with this. For dev/release builds, we don't, so I will fix.